pkgs/top-level/all-packages.nix

```diff
@@ -7144,7 +7144,7 @@ with pkgs;
 
   grabserial = callPackage ../development/tools/grabserial { };
 
-  mypy = callPackage ../development/tools/mypy { };
+  mypy = python3Packages.callPackage ../development/tools/mypy { };
```

Comment:
Do you mean you prefer to pass in individual requirements instead of a whole set? I can understand that. However, I really think we should make a clear separation between libraries, and applications. Otherwise, we may end up again with libraries in `all-packages.nix` and the trouble that that caused.